net-misc/eventd/eventd-0.23.0.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic linux-info meson ninja-utils systemd xdg-utils
+
+DESCRIPTION="A small daemon to act on remote or local events"
+HOMEPAGE="https://www.eventd.org/"
+SRC_URI="https://www.eventd.org/download/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ LGPL-3+ MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug fbcon +introspection ipv6 libcanberra libnotify +notification
+	pulseaudio purple speech systemd test upnp websocket +X zeroconf"
+
+REQUIRED_USE="
+	X? ( notification )
+	fbcon? ( notification )
+	notification? ( || ( X fbcon ) )
+	test? ( websocket )
+"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.40:2
+	sys-apps/util-linux
+	introspection? ( >=dev-libs/gobject-introspection-1.42 )
+	libcanberra? ( media-libs/libcanberra )
+	libnotify? ( x11-libs/gdk-pixbuf:2 )
+	notification? (
+		x11-libs/cairo
+		x11-libs/pango
+		x11-libs/gdk-pixbuf:2
+		X? (
+			x11-libs/cairo[xcb]
+			x11-libs/libxcb:=
+			x11-libs/xcb-util
+			x11-libs/xcb-util-wm
+		)
+	)
+	pulseaudio? (
+		media-libs/libsndfile
+		media-sound/pulseaudio
+	)
+	purple? ( net-im/pidgin )
+	speech? ( app-accessibility/speech-dispatcher )
+	systemd? ( sys-apps/systemd:= )
+	upnp? ( net-libs/gssdp:= )
+	websocket? ( >=net-libs/libsoup-2.50:2.4 )
+	zeroconf? ( net-dns/avahi[dbus] )
+"
+DEPEND="${COMMON_DEPEND}
+	app-text/docbook-xml-dtd:4.5
+	app-text/docbook-xsl-stylesheets
+	dev-libs/libxslt
+	virtual/pkgconfig
+	fbcon? ( virtual/os-headers )
+"
+RDEPEND="${COMMON_DEPEND}
+	net-libs/glib-networking[ssl]
+"
+
+eventd_check_compiler() {
```

Comment:
I will let you leave them for this time but next time, please really do get rid of this. The purpose of this mask was to get rid of all the checks. `pkg_pretend()` is slow, and doing meaningless checks for unsupported systems is a waste of everyone's time.